linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/ZipkinTracePropagator.scala

```diff
@@ -61,28 +58,121 @@ object ZipkinTrace {
   val ZipkinSampleHeader = "x-b3-sampled"
   val ZipkinFlagsHeader = "x-b3-flags"
 
+  /**
```

Comment:
Ideally we could share code between the http and h2 implementations.  There's a `router/base-http` project that is where code shared between `router/http` and `router/h2` lives.  We could do a similar thing here and create a `linkerd/protocol/base-http` project to hold code which can be shared by `linkerd/protocol/http` and `linkerd/protocol/h2`.

Reply:
Done, moved common code to linkerd/protocol/base-http and redone the whole implementation around it. See my other PR.